src/rust/engine/src/context.rs

```diff
@@ -33,6 +35,18 @@ use watch::{Invalidatable, InvalidationWatcher};
 
 const GIGABYTES: usize = 1024 * 1024 * 1024;
 
+// The reqwest crate has no support for ingesting multiple certificates in a single file,
```

Comment:
Is https://crates.io/crates/pem this the specific crate you mean? If so it might be good to clarify that in the comment.

Reply:
That is the one